_pkgdown.yml

```diff
@@ -4,17 +4,21 @@ template:
 
 reference:
 
-  - title: Main functions and datasets
+  - title: XCTR functions and datasets
     contents:
       - has_concept("XCTR functions")
-      - has_concept("XCTR datasets")
+
       - has_concept("ICTR datasets")
       - has_concept("PCTR datasets")
+      - has_concept("XCTR datasets")
 
+  - title: XSTR functions and datasets
+    contents:
       - has_concept("ISTR functions")
-      - has_concept("ISTR datasets")
-
       - has_concept("PSTR functions")
+
+      - has_concept("XSTR datasets")
+      - has_concept("ISTR datasets")
```

Comment:
Sorry I took the change to rearrange the website reference a bit. This doesn't quite belong to this PR but doesn't seem to interfere as it's such a different type of change.